### ISSUE TITLE
comment-tag doesn’t work anymore

### ISSUE THREAD

**Issue by hanspagel** [thollander/actions-comment-pull-request#397]
We just recently upgraded to thollander/actions-comment-pull-request@v3, since then comments are added again and again and the `comment-tag` is ignored. 🫨 We’re switching back to v2, but any idea what we’re doing wrong?

Example PR: https://github.com/scalar/scalar/pull/3236
Our GHA workflow: https://github.com/scalar/scalar/blob/main/.github/workflows/ci.yml#L288-L296

**Comment by koppor** [comment]
Underscore vs. dash issue - the README.md is inconsistent:

![image](https://github.com/user-attachments/assets/00de5e0d-95a2-4900-b5fa-12e02aefbe05)

![image](https://github.com/user-attachments/assets/b6e737db-d8ce-476c-b900-235b4a526093)

---

I would propose to update the README.md to consistently use the new dash (done at https://github.com/thollander/actions-comment-pull-request/pull/399) - or release a `v4` and reverting to underscore.

**Comment by nitrocode** [comment]
The title of the issue is a little confusing.

Is it that `comment_tag` input doesn't work anymore and the new input is `comment-tag` so if the new input is used, the issue is resolved ?

Maybe the old inputs can be added back in but as deprecated ?

```yaml
inputs:
  comment_tag:
    deprecationMessage: 'This option is removed. Use comment-tag instead.'
  comment-tag:
    description: 'A tag on your comment that will be used to identify a comment in case of replacement.'
```

https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddeprecationmessage

FYI introduced by

- PR https://github.com/thollander/actions-comment-pull-request/pull/391 to close #386

**Comment by koppor** [comment]
> Is it that `comment_tag` input doesn't work anymore and the new input is `comment-tag` so if the new input is used, the issue is resolved ?

Yes.

**Comment by ViacheslavKudinov** [comment]
Hi,
i think all the examples on READMe.md file have to be updated with new input names where is used `kebab-case` instead of current wrong ones with `snake_case`.

**Comment by thollander** [comment]
Hey, 

Thanks for the feedbacks. Indeed there was an issue in the refactoring around logs and documentation. 
Fixed in `v3.0.1` and also on `v3` (pointing to `v3.0.1`) ✅
